documentation/models.md

```diff
@@ -367,6 +367,141 @@ POST /_plugins/_ml/models/_register
+                                        "dataAsMap": {
+                                            "type": "object",
+                                            "properties": {
+                                                "id": {
+                                                    "type": "string"
+                                                },
+                                                "texts": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "string"
+                                                    }
+                                                },
+                                                "embeddings": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "array",
+                                                        "items": {
+                                                            "type": "number"
+                                                        }
+                                                    }
+                                                },
+                                                "meta": {
+                                                    "type": "object",
+                                                    "properties": {
+                                                        "api_version": {
+                                                            "type": "object",
+                                                            "properties": {
+                                                                "version": {
+                                                                    "type": "string"
+                                                                }
+                                                            }
+                                                        },
+                                                        "billed_units": {
+                                                            "type": "object",
+                                                            "properties": {
+                                                                "input_tokens": {
+                                                                    "type": "integer"
+                                                                }
+                                                            }
+                                                        }
+                                                    }
+                                                },
+                                                "response_type": {
+                                                    "type": "string"
+                                                }
```

Comment:
Suggest we set this to just return `embeddings` - the other data likely wouldn't be useful and don't need to over-expose the outputs.